api/app/Http/routes.php

```diff
@@ -32,15 +32,17 @@
 
 });
 
-$app->group(['prefix' => 'users'], function($app){
-
-    $app->get('/', 'App\Http\Controllers\UserController@getAll');
-
-    $app->get('/{id}', 'App\Http\Controllers\UserController@getOne');
+$app->get('timezones', function() { return App\Models\Timezones::getTimezones(); });
```

Comment:
This will need to go through a transformer as there are snake cased keys in the response.